eng/scripts/cleanup-allowlist.txt

```diff
@@ -10,3 +9,10 @@ AzSecPackAutoConfigRG
 
 static-test-resources
 LiveTestSecrets
+
+# Exclude child resource groups created by Azure resources.
+# These often have locks/permissions that explicitly deny being able to be manually deleted.
+# The groups get cleaned up when their parent group is deleted.
+# For example, Azure Synapse (synapseworkspace-managedrg-adaed0ea-df22-4ec1-9280-5ac85cf2f87a) and Azure Purview (managed-rg-mypurviewaccount).
+
+*managed*
```

Comment:
We should keep an eye on this to make sure it doesn't end up excluding more things then we want.